liferay-workspace/tests/blade.language.test/src/testFunctional/java/com/liferay/blade/samples/language/web/test/BladeLanguageWebTest.java

```diff
@@ -1,17 +1,15 @@
 /**
```

Comment:
Switching to LGPL was only to satisfy the SF. it complains if it sees Apache 2.0. Is that configurable?